From 50db443bad69fa7fe43cd06a53c8f462b5c82ad0 Mon Sep 17 00:00:00 2001
From: Junxiao Chang <junxiao.chang@intel.com>
Date: Tue, 29 Oct 2024 13:45:13 +0800
Subject: [PATCH 10/76] config: enable GSC and PXE for HuC support

dGPU needs MEI GSC/PXE to authenticate HuC firmware. Enable it
to support dGPU HuC firmware.

Signed-off-by: Junxiao Chang <junxiao.chang@intel.com>
---
 Intel/config    | 6 +++---
 Intel/config-rt | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Intel/config b/Intel/config
index a849f9ae6567d..bbfbc75780a15 100644
--- a/Intel/config
+++ b/Intel/config
@@ -2187,10 +2187,10 @@ CONFIG_EEPROM_93CX6=m
 CONFIG_INTEL_MEI=m
 CONFIG_INTEL_MEI_ME=m
 # CONFIG_INTEL_MEI_TXE is not set
-# CONFIG_INTEL_MEI_GSC is not set
+CONFIG_INTEL_MEI_GSC=m
 # CONFIG_INTEL_MEI_HDCP is not set
-# CONFIG_INTEL_MEI_PXP is not set
-# CONFIG_INTEL_MEI_GSC_PROXY is not set
+CONFIG_INTEL_MEI_PXP=m
+CONFIG_INTEL_MEI_GSC_PROXY=m
 CONFIG_VMWARE_VMCI=m
 # CONFIG_GENWQE is not set
 # CONFIG_ECHO is not set
diff --git a/Intel/config-rt b/Intel/config-rt
index fff0b80cd55af..3583bd57a17cb 100644
--- a/Intel/config-rt
+++ b/Intel/config-rt
@@ -2193,10 +2193,10 @@ CONFIG_EEPROM_93CX6=m
 CONFIG_INTEL_MEI=m
 CONFIG_INTEL_MEI_ME=m
 # CONFIG_INTEL_MEI_TXE is not set
-# CONFIG_INTEL_MEI_GSC is not set
+CONFIG_INTEL_MEI_GSC=m
 # CONFIG_INTEL_MEI_HDCP is not set
-# CONFIG_INTEL_MEI_PXP is not set
-# CONFIG_INTEL_MEI_GSC_PROXY is not set
+CONFIG_INTEL_MEI_PXP=m
+CONFIG_INTEL_MEI_GSC_PROXY=m
 CONFIG_VMWARE_VMCI=m
 # CONFIG_GENWQE is not set
 # CONFIG_ECHO is not set
-- 
2.43.0

